codemetrics.el

```diff
@@ -255,10 +256,14 @@ details.  Optional argument DEPTH is used for recursive depth calculation."
   (codemetrics--ensure-ts
     (let* ((mode (or mode major-mode))
            (rules (codemetrics--rules mode))
-           (nested-level)  ; this is the "starting" nested level
+           ;; Collection of nesting levels
+           (nested-depths '())
            (nested 0)
            (score 0)
            (data)
+           ;; Helper for calculating nested value from our collection of nestings
+           (calculate-nested-value (lambda (nested-depths)
+                                     (max 0 (- (length nested-depths) 1))))
```

Comment:
We can use `(1- )` here.

Reply:
Nice! Looked for that function, but for some reason I remembered the name as `-1` 😆 Changed it now in latest commit.